### PR TITLE
Clean up content

### PR DESCRIPTION
**Pull request by bzg**
Fix https://github.com/opendatateam/getudata.org/issues/7

### CODE REVIEW

**Comment by bzg** [comment]
@noirbizarre can you review this issue? There should be nothing problematic here. Thanks!

**Comment by bzg** [comment]
Hello @noirbizarre @abulte, can someone review this PR?

**Review by taniki — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,5 +2,11 @@
 
 [![CircleCI](https://circleci.com/gh/opendatateam/getudata.org/tree/master.svg?style=svg)](https://circleci.com/gh/opendatateam/workflows/getudata.org)
 
-udata official website
+udata is an open source and customizable data portal and catalog aimed
```
Comment (taniki) [inline README.md]: Il manque une majuscule à cause du début de phrase.

Comment (bzg) [reply README.md]: C'est fait. Je merge ?

**Comment by bzg** [comment]
https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2Fmedia.giphy.com%2Fmedia%2F2aAcLrYtiX8YM%2Fgiphy.gif&f=1

**Comment by bzg** [comment]
youpi

Done.

File: publishconf.py [hunk 2]
```diff
@@ -32,3 +32,5 @@
 
 PIWIK_URL = 'stats.data.gouv.fr'
 PIWIK_SITE_ID = 60
+
+DISPLAY_PAGES_ON_MENU = False
```
Comment (noirbizarre) [inline publishconf.py:36]: Seems to have broken the header: see https://getudata.org/

Comment (bzg) [reply publishconf.py:36]: Do you want to fix it or shall I give it a try?

**Pull request merged**